src/Extension/Propagator/XCloudTrace/XCloudTraceFormatter.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Copyright 2017 OpenCensus Authors
```

Comment:
@bobstrecansky I'm not sure about putting external copyright notices in CNCF code. Do you have any thoughts here?
If the code is just "inspired by" then we've done things like annotate an `@see other-repo`...

Reply:
I believe that is just a copy paste artifact from the OpenCensus repo.  Can we remove it?

Reply:
I'm not sure what the implications are of copying the code from OpenCensus, and whether this notice should be preserved. This class DOES have modifications, most notable line 80 (the `str_pad`), but I can remove the notice.

Reply:
Updated, pushed PR.